src/components/vm/disks/diskAdd.tsx

```diff
@@ -519,23 +657,19 @@ export const AddDiskModalBody = ({ disk, idPrefix, isMediaInsertion, vm, vms, su
             }));
             break;
         }
-        case 'mode': {
-            setMode(value);
-            break;
-        }
```

Comment:
Quo vadis?

Reply:
`setMode` is now called directly instead of indirectly via `onValueChanged("mode", ...)`.